backend/src/test/java/com/google/sps/TimeTest.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps;
+
+import com.google.sps.data.Time;
+import java.util.Date;
+import java.util.Calendar;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+public final class TimeTest {
+
+  private Time time;
+
+  @Before
+  public void setUp() {
+    // Set "current" date to  1/1/2020 2:00pm GMT
+    Calendar c = Calendar.getInstance();
+    c.set(2020, 0, 1, 14, 0, 0);
+    c.set(Calendar.MILLISECOND, 0);
+    Date date = c.getTime();
+
+    time = new Time(date);
+  }
+
+  @Test
+  public void inputTimeToMillis() {
```

Comment:
Since the getTimeMillis() method has two edge cases that check for out of bound hours/minutes, would it be possible to add tests that ensure an IllegalArgumentException is thrown?

Reply:
+1 to add tests covering corner cases.

Reply:
Added tests, thanks for the suggestion!